apps/frontend/src/libs/components/player-track/player-track.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+	MILLISECONDS_IN_SECOND,
+	PERCENT_MULTIPLIER,
+	START_TIME,
+} from "@meetlytic/shared";
```

Comment:
You cannot import directly from shared to component. Check how we import from shared in other places